docs/api/js/accounts-l1-l2.md

```diff
@@ -28,12 +28,12 @@ async approveERC20(
 
 ### Inputs and outputs
 
-| Name                 | Description                                                                      |
-| -------------------- | -------------------------------------------------------------------------------- |
-| token                | The Ethereum address of the token.                                               |
-| amount               | The amount of the token to be approved.                                          |
-| overrides (optional) | Ethereum transaction overrides. May be used to pass `gasLimit`, `gasPrice`, etc. You can also provide a custom address of the L1 bridge to use (the bridge provided by the Matter Labs team is used by default). |
-| returns              | `ethers.providers.TransactionResponse` object.                                   |
+| Name                 | Description                                                                                                                                                                                                      |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| token                | The Ethereum address of the token.                                                                                                                                                                               |
+| amount               | The amount of the token to be approved.                                                                                                                                                                          |
+| overrides? | **Ethereum** transaction overrides. May be used to pass `gasLimit`, `gasPrice`, etc. You can also provide a custom address of the L1 bridge to use (the bridge provided by the `Matter Labs` team is used by default). |
```

Comment:
@StanislavBreadless  what do you think?
since it is **js** SDK it is clean from the context what is `?`
 writing `(optional)` everywhere, looks ugly

Reply:
@Deniallugo @AnastasiiaVashchuk TBH, either option looks fine to me. Maybe it is better to keep `?` for consiceness

Reply:
conciseness is bad when we are talking about documentation. 
I have no strong opinion in this case. 
But I'd explicit documentation, rather than implicit

Reply:
Put my 5 cents, in this case conciseness  looks good, but in general I agree with @Deniallugo